app/__init__.py

```diff
@@ -1,7 +1,20 @@
 from flask import Flask
+from flask_login import LoginManager
+from app.src.repositories import UsersRepository
 
 app = Flask(__name__)
 app.secret_key = 'Tomibomi'
 
+
+login_manager = LoginManager()
+login_manager.init_app(app)
+
+
+@login_manager.user_loader
+def load_user(user_id):
```

Comment:
that code has to be in init?

Reply:
I don't know, where I should put it ?

Reply:
it is retrieving user from repository why is it in init not in some file related to user or that login manager

Reply:
add types